.eslintrc.cjs

```diff
@@ -21,6 +21,10 @@ module.exports = {
     {
       files: ['**/*.comp.cy.js', '**/*.unit.cy.js', '**/*.e2e.cy.js'],
       extends: ['plugin:cypress/recommended'],
+      rules: {
+        'cypress/no-force': 'error',
```

Comment:
`no-force` does not check for `.only.`. Looks like https://www.npmjs.com/package/eslint-plugin-no-only-tests is a good solution.  Will need to npm install it to the dev and then add the rule.  Instructions are on the linked page.